src/ExplorerRoutes.tsx

```diff
@@ -35,6 +36,10 @@ export default function ExplorerRoutes() {
           <Route path=":struct" element={<CoinPage />} />
           <Route path=":struct/:tab" element={<CoinPage />} />
         </Route>
+        <Route path="/fungible_asset">
+          <Route path=":address" element={<FAPage />} />
+          <Route path=":address/:tab" element={<FAPage />} />
+        </Route>
```

Comment:
out of scope but do you think we may add a assets page to explorer? reference https://aptoscan.com/coins

Reply:
That's the last thing I may add here, though I'd have to check on the efficiency of the indexer queries

Reply:
@0xbe1 please open a GitHub issue for the coin page.